numismatic/feeds/poloniex.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import json
+import time
+from datetime import datetime
+
+from streamz import Stream
+import attr
+import websockets
+
+from .base import Feed, WebsocketClient, STOP_HANDLERS
+from ..events import Heartbeat, Trade, Order
+
+logger = logging.getLogger(__name__)
+
+class PoloniexWebsocketClient(WebsocketClient):
+    '''
+        Poloniex Websocket client. This uses an
+        undocumented websocket_url. The alternative
+        is using Poloniex's WAMP protocol. 
+    '''
+    exchange = 'Poloniex'
+    websocket_url = 'wss://api2.poloniex.com/'
+
+    async def _subscribe(self, subscription):
+        await super()._subscribe(subscription)
+
+        # Poloniex requires format USDT_<currency symbol>
+        # when comparing a USD to a coin
+        channel = subscription.channel
+        if subscription.symbol[-4:] == '-USD':
+            channel = f'USDT_{subscription.symbol[:3]}'
```

Comment:
Get pairs is not generic enough. It returns asset_currency, whereas I need currency_asset. But I think I should make it more generic though

Reply:
The reason this is a method and not `asset+currency` is so that different Feeds can override it. Set it to whatever it needs to be for Poloniex. Look for example at GDAX which I think uses a `-` separator.